src/GitLink/Extensions/RepositoryExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace GitLink
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Catel;
+    using Catel.IO;
+    using LibGit2Sharp;
+
+    internal static class RepositoryExtensions
+    {
+        internal static string GetRepoNormalizedPath(this Repository repository, string path)
+        {
+            Argument.IsNotNull(nameof(repository), repository);
```

Comment:
Depends, I still have to do 2 names, where () => repository only requires one. But again, just a matter of preference.